src/components/results/financial-risk-results/personality-score-bar/index.js

```diff
@@ -19,27 +27,38 @@ class PersonalityScoreBar extends Component {
   }
   render() {
     if(!this.props.isReady("results")) { return null; }
```

Comment:
I think we'd also want to return in the case that there is no archetype, similar to how the other financial risk components do

Reply:
Everything else looks good though